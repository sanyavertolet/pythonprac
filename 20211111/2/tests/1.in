1, 2, 3, 4, 5, 6
data
wrong data
(1, 1), (2, 2), (3, 3)
(0, 0), (0, 1), (1, 0)
