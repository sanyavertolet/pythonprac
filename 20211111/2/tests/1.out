Invalid input
Invalid input
Invalid input
Not a triangle
0.50
